package.json

```diff
@@ -8,8 +8,14 @@
     "test": "jest"
   },
   "dependencies": {
+    "apollo-server": "^2.9.9",
+    "apollo-server-testing": "^2.9.7",
```

Comment:
`apollo-server-testing` should be a `devDependency` as you don't need it at runtime, only for development.